articles/Setup-Windows-Python.md

```diff
@@ -167,17 +191,17 @@ Depending on the Python and CNTK version (CPU or GPU) we supply different wheel
 
 ---
 
-### Anaconda3
+#### Anaconda3
 
 We have been testing CNTK with Anaconda3 4.1.1 (64-bit) and Python versions 2.7 and 3.5, as well as Anaconda3 4.3.1 with Python version 3.6. If you do not have a Anaconda3 Python installation, install [Anaconda3 4.1.1 Python for Windows (64-bit)](https://repo.continuum.io/archive/Anaconda3-4.1.1-Windows-x86_64.exe).
 
-Below we assume Anaconda is installed and that it is listed before any other Python installations in your PATH. If you plan on using a GPU enabled version of CNTK, you will need a CUDA 9 compliant graphics card and up-to-date graphics drivers installed on your system.
+Below we assume Anaconda is installed and that it is listed before any other Python installations in your PATH. If you plan on using a GPU-enabled version of CNTK, you will need a CUDA 9 compliant graphics card and up-to-date graphics drivers installed on your system.
```

Comment:
Recently users are reporting bugs and the root cause is that they are installing CUDA 9.1 instead of CUDA 9.0. Maybe we should point this out